addon/webextension/background/auth.js

```diff
@@ -157,6 +157,19 @@ window.auth = (function () {
     });
   };
 
+  exports.ensureLoggedIn = function () {
+    return browser.cookies.get({
```

Comment:
they both just call login(), so don't they have to act the same? – it's true it can't handle explicitly clearing cookies after login occurred, but that seems like a fairly obscure case, and the lazy login in the other PR should handle that situation anyway.  Unless there's something I'm missing?